components/esp_dns/README.md

```diff
@@ -0,0 +1,223 @@
+# ESP DNS Component
+
+This component provides a flexible DNS resolution system for ESP32 devices with support for multiple DNS protocols. It allows applications to securely resolve domain names using various transport methods, including standard UDP/TCP DNS, DNS over TLS (DoT), and DNS over HTTPS (DoH).
```

Comment:
to **securely** resolve domain names... ...standard UDP/TCP DNS
 
 is this secure?

Reply:
You're right, plain TCP/UDP DNS isn't secure. I've updated the text to reflect that.